eitprocessing/datahandling/sequence.py

```diff
@@ -56,7 +58,7 @@ def time(self) -> np.ndarray:
         if len(self.eit_data):
             return self.eit_data["raw"].time
         if len(self.continuous_data):
-            return next(self.continuous_data.values())
+            return next(iter(self.continuous_data.values()))
```

Comment:
next() is required because the `values()` view is not indexable (because technically dicts don't have an order). By wrapping it in an iter() you can grab the first item with next(). It does roughly the same as `list(self.continuous_data.values())[0]` without creating the entire list.

Reply:
This method never returns None. It either returns an array or raises an exception.